lib/fetch/request.js

```diff
@@ -127,12 +127,12 @@ class Request {
     }
 
     // 10. If init["window"] exists and is non-null, then throw a TypeError.
-    if ('window' in init && window != null) {
+    if (init.window !== undefined && window != null) {
```

Comment:
shouldn't this have `init.window != null` ?

Reply:
@chlonel can you fix?

Reply:
Actually it doesn't matter here. It's the same.

Reply:
It's not. `window` is a local variable created a few lines above (`let window = 'client'`).

Reply:
> If init["window"] exists and is non-null, then throw a TypeError.

To me, this means that we should check if `init["window"]` in non-null, not `window`. But maybe the spec or the comment is wrong.

Reply:
Ah, sorry I missed that

Reply:
[Fixed](https://github.com/nodejs/undici/commit/ec2813498dac2534a53590b5e81d92c3ce334597)